compass-custom-archetype/src/test/resources/projects/basic/archetype.properties

```diff
@@ -3,3 +3,4 @@ package=it.pkg
 version=0.1-SNAPSHOT
 groupId=archetype.it
 artifactId=basic
+compassVersion=1.0-SNAPSHOT
```

Comment:
AFAICS, this is used by maven when test-instantiating the archetype. It would probably be most correct if this were `${project.version}`, i.e. the current COMPASS version, as well. I'll see if I can make test resource filtering do the right thing.